packages/demand-bloc-quotes/src/QuotesBloc.ts

```diff
@@ -1,14 +1,14 @@
 import {
-  QuotesV2,
-  QuotesV2SearchParams,
   errorCodes,
-  QuotesV2Response,
   HttpResponse,
+  QuotesV2,
+  QuotesV2Response,
+  QuotesV2SearchParams,
 } from '@karhoo/demand-api'
 import { Subject, Subscription, timer } from 'rxjs'
-import { publishReplay, refCount, map, distinctUntilChanged } from 'rxjs/operators'
+import { distinctUntilChanged, map, publishReplay, refCount } from 'rxjs/operators'
 import { poll } from './polling'
-import { transformer, QuoteItem } from './transformer'
+import { QuoteItem, transformer } from './transformer'
```

Comment:
How come the order of these imports has changed? Are you using different prettifier rules?

Reply:
Truly I have no idea 🤷 
I will revert that back.

Reply:
I haven't touched the linting rules. I checked my vscode extension but could not see anything relevant. I try changing it but it comes back as soon as.I press save. I could use another IDE or something like vim, but I don't really see the point.